app/src/bootstrap.ts

```diff
@@ -1,18 +1,21 @@
+import PlayerHandler from './handlers/PlayerHandler';
+
 require ('reflect-metadata');
 
 import { Container } from 'inversify';
 import Kernel from './bootstrap/Kernel';
 import config from './config';
+import PlayerHandlerInterface from './interfaces/PlayerHandlerInterface';
 
 // Initialize container
 const app = new Container({ autoBindInjectable: true, skipBaseClassChecks: true});
 
 // Bind container to app const
 app.bind<Container>(Container).toConstantValue(app);
-
 // Bind Kernel to the application
 app.bind<Kernel>(Kernel).toSelf().inSingletonScope();
 
+app.bind<PlayerHandlerInterface>('PlayerHandlerInterface').to(PlayerHandler).inSingletonScope();
```

Comment:
You should register this in a ContainerModule like https://github.com/marci4/ps2alert-websocket-dev/blob/features/18/app/src/services/census/index.ts.

Also might be better to register a Symbol for the interface and use that as ServiceIdentifier instead of a string.